### ISSUE TITLE
Editorial Flow and policies

### ISSUE THREAD

**Issue by mtilburgadobe** [hlxsites/fcbayern#38]

**Comment by cwoeltge** [comment]
I would actually think this is a YAGNI. Publish button will become more complex, the user has to go in later and publish again, might not understand why "it doesn't work". Technically it also requires dependend logic between documents which I would not put to the publish process but rather as part of index/navigation/display logic.

@davidnuescheler opinions?

**Comment by mtilburgadobe** [comment]
Also adding @rofe , as it was his suggestion... open to discuss of course!

> . Technically it also requires dependend logic between documents which I would not put to the publish process but rather as part of index/navigation/display logic.

If I understand the requirement correctly, the dependency is not on the content existing, it's that the content is published (live) correct?

**Comment by cwoeltge** [comment]
yes. DE content is leading and mandatory. any information must be available in DE first. translated content must thus for follow. 
one way is to prevent publishing without related german content. another way would be to check before it appears (navigation/search/display)

**Comment by mtilburgadobe** [comment]
I see your point, counter point -> in the suggested model, it would allow for translated content to be published before german.  Okay it's hidden, but this would automatically become visible when the german content is published.  Essentially this means that publishing german is publishing also english, which might also be confusing (and possibly problematic if the english page needs edits)

What about in the sidekick, if the german edition is not there, and we don't publish the translated version, we re-direct the page to something that explains that the content is not published.  This could be a static page somewhere, even internal(possibly github readme)

Or if it's possible to have the sidekick generate an alert box with the info?

**Comment by mtilburgadobe** [comment]
> Or if it's possible to have the sidekick generate an alert box with the info?

Yes it is... this could be an example(of course we work on the wording/translation etc)
<img width="448" alt="Screenshot 2023-05-12 at 05 27 27" src="https://github.com/hlxsites/fcbayern/assets/77688257/12c25323-f54b-4182-9d61-2484038a62db">
